var/spack/repos/builtin/packages/tinyxml2/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+
+class Tinyxml2(Package):
```

Comment:
use `CMakePackage`, see `trilinos` for example. For this particular package, you would only change debug/release build (see `dealii` on how to do it).